azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/telemetry/AppInsightsProxy.java

```diff
@@ -24,15 +36,48 @@
     protected boolean isEnabled = true;
 
     public AppInsightsProxy(final TelemetryConfiguration config) {
-        // InstrumentationKey will be read from ApplicationInsights.xml
-        client = new TelemetryClient();
+        client = new TelemetryClient(readConfigurationFromFile());
         if (config == null) {
             throw new NullPointerException();
         }
         configuration = config;
         defaultProperties = configuration.getTelemetryProperties();
     }
 
+
+//      This is a workaround for telemetry issue. ApplicationInsight read configuration file by JAXB, and JAXB parse
+//      configuration by JAXBContext, but the context model differs in Java 8 and Java 11 during maven execution, so
```

Comment:
Attach the issue in SDK, and mention that this code is legacy(using regex) will be deleted when referenced issue is resolved by SDK team.

Reply:
Add the link and mark it will be remove in future.